src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.CognitiveServices.Search.LocalSearch</PackageId>
+		<Description>Provides ability to search for businesses</Description>
+		<Version>0.9.0-preview</Version>
+		<AssemblyName>Microsoft.Azure.CognitiveServices.BingLocalSearch</AssemblyName>    
+		<PackageTags>CognitiveServices;Bing;BingLocalSearch;LocalSearch;Local Search</PackageTags>
+		<PackageReleaseNotes>
+			<![CDATA[
+                >> put release notes here
+      ]]>
+		</PackageReleaseNotes>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+	</PropertyGroup>
+
+	<!-- Please do not move/edit code below this line -->
+	<Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
+	<ItemGroup>
```

Comment:
delete line 25 to 28

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/4962/commits/821162637d4dc9db162df81b71a02abf6b974fae